contentcuration/contentcuration/management/commands/mark_incomplete.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Exists
+from django.db.models import OuterRef
+from django.db.models import Q
+from le_utils.constants import content_kinds
+from le_utils.constants import exercises
+
+from contentcuration.models import AssessmentItem
+from contentcuration.models import ContentNode
+from contentcuration.models import File
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        exercise_check_query = AssessmentItem.objects.filter(contentnode=OuterRef('id')) \
+            .exclude(type=exercises.PERSEUS_QUESTION)\
+            .filter(
+                Q(question='') |
+                Q(answers='[]') |
+                (~Q(type=exercises.INPUT_QUESTION) & ~Q(answers__iregex='"correct":true'))  # hack to check if no correct answers
```

Comment:
Am I right that it's possible to use `~Q(answers__iregex='"correct":true'))` because we store answers as a string?

Reply:
If yes, it might be good to add a mention about this line to this issue https://github.com/learningequality/studio/issues/1855, because I guess it would stop work after we address the issue mentioned

Reply:
Yes, I'll make a note of it, but hopefully we'll run this one time so we can just get rid of this command before we fix that issue